src/app.controller.get.ts

```
@@ -537,6 +539,20 @@ export class GetController {
 			})
 		}
 
+		// Check if we're using the data cache and if so, if we can use it
+		if (this.configService.get<boolean>('USE_DATA_CACHING')) {
+			
+			const cachedResult = await this.dataCache.get({
```

ESLint

[error] Delete `⏎↹↹↹`

(prettier/prettier)